### PR TITLE
Create tests for Mapper package from 16 to 36 points without 26 point

### PR DESCRIPTION
**Pull request by StannyB9**

### CODE REVIEW

**Review by DZotsenko — COMMENTED** [review]
Would it be better to remove pom.xml changes from the commit?

**Review by Andrii-Danylenko — APPROVED** [review]
Everything is fine, except for the redundant pom.xml. Please exclude it from the next commit.

**Review by DenisOgr — APPROVED** [review]
From my side - OK.
New line in POM file - also OK

File: service/src/test/java/greencity/mapping/NewTagDtoMapperTest.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import org.junit.jupiter.api.Test;
+import org.modelmapper.ModelMapper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class NewTagDtoMapperTest {
+    private NewTagDtoMapper newTagDtoMapper;
```
Comment (maks741) [inline service/src/test/java/greencity/mapping/NewTagDtoMapperTest.java:18]:
```suggestion
    private final NewTagDtoMapper newTagDtoMapper = new NewTagDtoMapper();
```

File: service/src/test/java/greencity/mapping/NewTagDtoMapperTest.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+    @BeforeEach
+    public void setUp() {
+        ModelMapper modelMapper = new ModelMapper();
+        newTagDtoMapper = new NewTagDtoMapper();
+        modelMapper.addConverter(newTagDtoMapper);
+    }
```
Comment (maks741) [inline service/src/test/java/greencity/mapping/NewTagDtoMapperTest.java:25]: redundant

**Review by maks741 — APPROVED** [review]

**Pull request merged**